### PR TITLE
feat(table): add empty table state

### PR DESCRIPTION
**Pull request by ej9x**

### CODE REVIEW

File: src/atoms/Table/Table.stories.js [hunk 1]
```diff
@@ -281,11 +281,51 @@ export default (asStory) => {
...
+              <Table.HeaderCell>Id</Table.HeaderCell>
+              <Table.HeaderCell>Name</Table.HeaderCell>
+              <Table.HeaderCell>Email</Table.HeaderCell>
+              <Table.HeaderCell>Created At</Table.HeaderCell>
+              <Table.HeaderCell>Updated At</Table.HeaderCell>
+              <Table.HeaderCell />
+            </Table.Header>
+
+            <Table.Body
+              actionTitle={ (
```
Comment (zouxuoz) [inline src/atoms/Table/Table.stories.js]: @ej9x may be rename `actionTitle` to `action` or something else? `actionTitle` with passed button looks strange

File: src/atoms/typography/Icon/glyphs/index.js [hunk 2]
```diff
@@ -43,6 +44,7 @@ export {
   CloudDatabase,
   Delete,
   Diagram,
+  DismissData,
```
Comment (zouxuoz) [inline src/atoms/typography/Icon/glyphs/index.js]: @ej9x why `dismiss`? may be `NoData`?

File: src/atoms/Table/TableBody.js [hunk 3]
```diff
@@ -20,33 +26,78 @@ const theme = createTheme(name, {
...
-      </AsyncContent>
-    </TableBodyTag>
-  );
+
+class TableBody extends PureComponent<TableBodyProps> {
+
+ hasBodyRows = (children: React$Node) => {
+   let hasBodyRows = false;
+
+   React.Children.forEach(children, child => {
```
Comment (zouxuoz) [inline src/atoms/Table/TableBody.js]: @ej9x strange check, I think we can delete ability to pass `children` to `TableBoby`, only `data`, after it we can delete this check too

File: src/atoms/Table/TableBody.js [hunk 4]
```diff
@@ -20,33 +26,78 @@ const theme = createTheme(name, {
...
+      child.type.prototype instanceof TableBodyRow
+     ) {
+       hasBodyRows = true;
+     }
+   });
+
+   return hasBodyRows;
+ };
+
+  renderActionTitle = () => {
```
Comment (zouxuoz) [inline src/atoms/Table/TableBody.js]: @ej9x why `renderActionTitle` if it's action component, not title?

File: src/atoms/typography/Icon/glyphs/index.js (hunk 2 above)
Comment (ej9x) [reply src/atoms/typography/Icon/glyphs/index.js]: I got this icon from the icons pack. It named there like DismissData.

File: src/atoms/Table/TableBody.js (hunk 3 above)
Comment (ej9x) [reply src/atoms/Table/TableBody.js]: why? ability to pass `children` looks useful.

File: src/atoms/Table/TableBody.js (hunk 4 above)
Comment (ej9x) [reply src/atoms/Table/TableBody.js]: fixed

File: src/atoms/Table/Table.stories.js (hunk 1 above)
Comment (ej9x) [reply src/atoms/Table/Table.stories.js]: fixed

**Pull request merged**